src/app/zap-templates/zcl/chip-devices.xml

```diff
@@ -186,6 +186,8 @@ limitations under the License.
       </include>
       <include cluster="Temperature Measurement" client="false" server="true" clientLocked="true" serverLocked="true">
       </include>
+      <include cluster="Network Provisioning" client="true" server="true" clientLocked="true" serverLocked="true">
```

Comment:
I think we don't want the client here but that's a matter of which patch lands first between this one and #4661... One of us will have to rebase some work :)

Reply:
The network provisioning cluster on client will accepts *Response commands, I have no idea what is the meaning of "client="true"", "server="true"" here.

Reply:
> The network provisioning cluster on client will accepts *Response commands, I have no idea what is the meaning of "client="true"", "server="true"" here.

My understanding is that those fields are to enable/disable a cluster by default for a particular device configuration. I just disable all client clusters from the `all-clusters-app.zap` file. I did a quick look at this file and most `client` are true by default.

So I guess it does not matter too much what the value of the field in this file is for this patch. We will need to take a look at that in a different issue.

That said. What you want to do in `.zap` files is:
 * Enable the server side of the `Network Provisioning` cluster in `examples/all-clusters-app/all-clusters-common/all-clusters-app.zap`
 * Enable the client side of the `Network Provisioning cluster` in `examples/chip-tool/chip-tool.zap`
 *  Enable the client side of the `Network Provisioning cluster` in `src/darwin/Framework/CHIP/chip-tool.zap`

So this way you will have 2 client applications and one server application.

Reply:
Done